zsh.sls

```diff
@@ -0,0 +1,8 @@
+zsh:
```

Comment:
this is a chicken-and-egg problem. the pkg.install doesn't work when bash is the shell.  ZSH does come installed by default..  We may just need to run the chsh in the bootstrap.

Reply:
I moved zsh installation to bin/bootstrap